### PR TITLE
added Last Modified to the App object, displayed in the list cell and…

### PR DESCRIPTION
**Pull request by vlondon**
… on the app details page

### CODE REVIEW

**Comment by vlondon** [comment]
this should resolve #47

**Comment by RishabhTayal** [comment]
Thanks for your pull request. Please add an entry in the [CHANGELOG.md](https://github.com/RishabhTayal/ReviewMonitor/blob/master/CHANGELOG.md)

File: ReviewMonitor/Apps List View Controller/AppsListTableViewCell.swift [hunk 1]
```diff
@@ -26,5 +27,13 @@ class AppsListTableViewCell: UITableViewCell {
...
         }
         appNameLabel.text = app.name
+        updateLastModifiedDate(from: app.lastModified.doubleValue)
+    }
+
+    private func updateLastModifiedDate(from timestamp: Double) {
+        let date = Date(timeIntervalSince1970: timestamp / 1000)
+        let formatter = DateFormatter()
+        formatter.dateFormat = "MMM dd, yyyy"
+        lastModifiedLabel.text = formatter.string(from: date)
```
Comment (RishabhTayal) [inline ReviewMonitor/Apps List View Controller/AppsListTableViewCell.swift]: Use https://github.com/RishabhTayal/ReviewMonitor/blob/53e278657d1933d6b639a17bd7525d9db8e359c1/ReviewMonitor/AppHelper.swift#L29

File: ReviewMonitor/Apps List View Controller/App Detail View Controller/AppDetailViewController.swift [hunk 2]
```diff
@@ -60,6 +62,13 @@ class AppDetailViewController: UIViewController {
         }
     }
 
+    private func updateLastModifiedDate(from timestamp: Double) {
+        let date = Date(timeIntervalSince1970: timestamp / 1000)
+        let formatter = DateFormatter()
+        formatter.dateFormat = "MMM dd, yyyy"
```
Comment (RishabhTayal) [inline ReviewMonitor/Apps List View Controller/App Detail View Controller/AppDetailViewController.swift]: Use: https://github.com/RishabhTayal/ReviewMonitor/blob/53e278657d1933d6b639a17bd7525d9db8e359c1/ReviewMonitor/AppHelper.swift#L29

Comment (vlondon) [reply ReviewMonitor/Apps List View Controller/App Detail View Controller/AppDetailViewController.swift]: done

**Comment by RishabhTayal** [comment]
Merged! Thanks for your awesome contribution!

**Pull request merged**